Assignment5/app.py

```
@@ -0,0 +1,184 @@
+@app.errorhandler(400)
+def bad_request(error):  # type: ignore[override]
+    return jsonify({"error": "Bad request"}), HTTPStatus.BAD_REQUEST
+
+
+@app.errorhandler(500)
+def server_error(error):  # type: ignore[override]
+    return jsonify({"error": "Internal server error"}), HTTPStatus.INTERNAL_SERVER_ERROR
+
+
+if __name__ == "__main__":  # pragma: no cover
+    app.run(debug=True)
```

Ruff (0.14.4)

Use of `debug=True` in Flask app detected

(S201)